build/hack/loadtest.go

```diff
@@ -0,0 +1,49 @@
+package hack
+
+import (
+	"context"
+
+	"dagger.io/dagger"
+	"github.com/google/uuid"
+)
+
+func LoadTest(ctx context.Context, client *dagger.Client, base, flipt *dagger.Container) error {
+	seed := base.File("build/testing/integration/readonly/testdata/default.yaml")
+	importCmd := []string{"/flipt", "import", "--create-namespace", "import.yaml"}
+
+	// import some test data
+	_, err := flipt.WithEnvVariable("UNIQUE", uuid.New().String()).
+		WithFile("import.yaml", seed).
+		WithExec(importCmd).
+		ExitCode(ctx)
+
+	if err != nil {
+		return err
+	}
```

Comment:
I think this would technically result in the layer produced at 17 having the state in it.
However, the built instance of the `flipt` container you build down on line 34 is built from the reference of flipt passed into this function.

I think the following might work (if not we can do what I do in the other tests and run import in a foreground container with flipt as a running service and go through the API):
```suggestion
	// import some test data
	flipt, err = flipt.WithEnvVariable("UNIQUE", uuid.New().String()).
		WithFile("import.yaml", seed).
		WithExec(importCmd).
		Sync(ctx)

	if err != nil {
		return err
	}
```

Reply:
thanks i wasnt sure what the difference was between `Sync` and `ExitCode` and when you would use one over the other

Reply:
much better!

![CleanShot 2023-07-16 at 09 22 27](https://github.com/flipt-io/flipt/assets/209477/1b6ce2ae-17e2-4341-aa84-d81b06265c0e)
![CleanShot 2023-07-16 at 09 19 34](https://github.com/flipt-io/flipt/assets/209477/e3b121d6-9ac8-44ef-a931-874b442594fa)

Reply:
Nice yeah that looks legit